pkg/store/google_secret_manager_store_test.go

```
@@ -0,0 +1,429 @@
+		return nil, args.Error(1)
+	}
+	return args.Get(0).(*secretmanagerpb.AccessSecretVersionResponse), args.Error(1)
+}
+
+func (m *MockGSMClient) Close() error {
+	args := m.Called()
+	return args.Error(0)
+}
+
+// Helper function to create a new mock client that satisfies the secretmanager.Client interface
+func newMockGSMClient() *secretmanager.Client {
```

golangci-lint (1.62.2)

func `newMockGSMClient` is unused

(unused)